bundles/org.openhab.binding.solarwatt/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+	https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:solarwatt:energymanager">
+		<parameter name="hostname" type="text">
+			<label>Host Name / IP Address</label>
+			<description>The host name/ip address of the solarwatt energymanager.</description>
+			<context>network-address</context>
+			<advanced>false</advanced>
```

Comment:
`false` is the default and can therefore be removed.

Reply:
I removed the flag.